setup.py

```diff
@@ -0,0 +1,14 @@
+from setuptools import setup, find_namespace_packages
+
+setup(name='console_bot',
+      version='1.1',
+      description='Console bot that will help you with everything',
+      url='https://github.com/TetianaFilonenko/project-team-4.git',
+      author='Kobzar Coders',
+      author_email='flyingcircus@example.com',
+      license='MIT',
+      packages= find_namespace_packages(),
+      install_requires= ['faker', 'prompt_toolkit'],
+      include_package_data=True,
+      entry_points= { 'console_scripts': ["run = console_bot:main"]}
```

Comment:
`run` is the command used in the terminal to call our app, right? If yes, plz rename it to something meaningful, e.g. `run-console-bot`